manifests/charts/ztunnel/templates/preemption.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
```

Comment:
Any prior art for doing this? Seems like other projects are just having users set it to system-node-critical and run in kube-system. Seems reasonable to do for us as well, I had assumed that is how ztunnel would be deployed